mobify/platform-scaffold
# Replace c classes

audit and replace `c-` classes with `pw-`

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1444

## Changes
- remove or replace `c-` classes to `pw-`

## Todos

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- make sure `c-` classes are only used in project's component
